src/manifold/src/csg_tree.cpp

```diff
@@ -478,9 +483,9 @@ std::shared_ptr<Manifold::Impl> CsgOpNode::BatchBoolean(
           continue;
         }
         group.run([&, a, b]() {
-          const Manifold::Impl *aImpl;
-          const Manifold::Impl *bImpl;
```

Comment:
Were these lines even doing anything before? Can we just delete them now too?

Reply:
sure